### DISCUSSION TITLE
Native support for qwik and/or astro

### DISCUSSION THREAD

**Discussion by zanettin** [nrwl/nx#12856] [Feature Requests]
next gen js frameworks like [astro](https://astro.build/) with its island approach and partial hydration or [qwik](https://qwik.builder.io/) with zero js and resumability instead of hydration are gaining momentum.

for astro there is already a community plugin but for qwik not. therefore my questions would be if you plan to add any further frameworks to the list of existing once in an upcoming major upgrade or do you plan to keep "just" the existing once?

https://npmtrends.com/@builder.io/qwik-vs-@remix-run/serve-vs-astro-vs-solid-js

**Comment by Brian-McBride** [comment]
Qwik is now supported in Astro directly, which can offer a path to supporting both.

**Comment by k3nsei** [comment]
There is already extension that allow to use nx and qwik https://github.com/qwikifiers/qwik-nx

**Comment by airtonix** [comment]
> for astro there is already a community plugin

Where?
